authentication/identity_providers/configuring-htpasswd-identity-provider.adoc

```diff
@@ -4,13 +4,34 @@ include::modules/common-attributes.adoc[]
 :context: configuring-htpasswd-identity-provider
 toc::[]
 
-Configure the `htpasswd` identity provider to validate user names and passwords
-against a flat file generated using
-link:http://httpd.apache.org/docs/2.4/programs/htpasswd.html[`htpasswd`].
+[id="identity-provider-overview-{context}"]
+= About identity providers in {product-title}
 
-include::modules/identity-provider-overview.adoc[leveloffset=+1]
+By default, only a `kubeadmin` user exists on your cluster. To specify an
+identity provider, you must create a Custom Resource (CR) that describes
+that identity provider and add it to the cluster.
 
-include::modules/identity-provider-creating-htpasswd-file.adoc[leveloffset=+1]
+[NOTE]
+====
+{product-title} user names containing `/`, `:`, and `%` are not supported.
+====
+
+To define an HTPasswd identity provider you must perform the
+following steps:
+
+. Create a `htpasswd` file to store the user and password information. 
+Instructions are found for
```

Comment:
Reads just a little funny to me. Maybe say "provided" instead of "found"?

Reply:
Adjusted to use `provided`.